apps/server/src/api-integration/integration.controller.ts

```diff
@@ -10,6 +10,11 @@ import { eventStore } from '../stores/EventStore.js';
 import * as assert from '../utils/assert.js';
 import { isEmptyObject } from '../utils/parserUtils.js';
 import { parseProperty, updateEvent } from './integration.utils.js';
+import { throttle } from '../utils/throttle.js';
+import { regenerateWhitelist } from '../services/rundown-service/rundownCacheUtils.js';
+
+const propertiesWhitelist = Object.freeze(Object.values(regenerateWhitelist));
```

Comment:
not sure we need the `Object.freeze`. This hasnt been something we use in the rest of the codebase
If, for some reason, we expect that this might be mutated, we can throw a typescript `Readonly` on it so we get told off :D 

You can check the current enum validation in the `validateEvent.tsx` file